x-pack/plugins/cloud_security_posture/public/pages/vulnerabilities/vulnerabilities.tsx

```diff
@@ -188,6 +204,7 @@ const VulnerabilitiesContent = ({ dataView }: { dataView: DataView }) => {
         <EmptyState onResetFilters={onResetFilters} />
       ) : (
         <>
+          {isFetching && <EuiProgress size="xs" color="accent" />}
```

Comment:
would be a bit smoother to save the space for this `EuiProgress` even when it's not rendered. would make it less jumpy. we can also set it as an `absolute` element to prevent that

Reply:
that was a great suggestion, I added a EuiSpacer with the same size as the EuiProgress